bitcoin/types_test.go

```diff
@@ -96,3 +57,12 @@ func Test_Block_UnmarshalJSON(t *testing.T) {
 		})
 	}
 }
+
+// loadTypeFixture takes a file name and returns the type fixture.
+func loadTypeFixture(fileName string) string {
+	content, err := ioutil.ReadFile(fmt.Sprintf("type_fixtures/%s", fileName))
```

Comment:
nit: inline declaration please per #47

Reply:
Hm. It looks like the CI format job doesn't like the appended `if` variant of inline declaration shown in the linked PR. I'm familiar with and quite enjoy the leading `if` variant, but I haven't encountered this one before. Thoughts?

Reply:
Hmm.. Yeah clearly it's failing because of that inline statement! I'm baffled now considering I've ran code with these inline statements w/o fail... It would be nice to get some decent ouput from that CI step haha.

TBH I lean towards the side of _"let's not add work around adjusting our linter"_ ; there are more important (fun) things to focus on! :)  (ie; imo let's ditch the style suggestion)

Pat opened the `[style]` issue from my comment, curious on his thoughts? (cc #47). 

- - -

Check out the [relevant excerpt](https://github.com/golang/go/wiki/CodeReviewComments#indent-error-flow) from the go-wiki on code style. 

> If the if statement has an initialization statement, such as:
> ```
> if x, err := f(); err != nil {
>	// error handling
>	return
> } else {
>	// use x
> }
>```
> then this may require moving the short variable declaration to its own line:
> ```
> x, err := f()
> if err != nil {
> 	// error handling
> 	return
> }
> // use x
> ```

Reply:
Thanks for adding the excerpt. I was able to reproduce this locally with `gofmt`.

```shell
$ gofmt -s -l -d .
bitcoin/types_test.go
diff -u bitcoin/types_test.go.orig bitcoin/types_test.go
--- bitcoin/types_test.go.orig	2021-05-08 10:32:43.000000000 -0400
+++ bitcoin/types_test.go	2021-05-08 10:32:43.000000000 -0400
@@ -60,7 +60,8 @@

 // loadTypeFixture takes a file name and returns the type fixture.
 func loadTypeFixture(fileName string) string {
-	content, err := ioutil.ReadFile(fmt.Sprintf("type_fixtures/%s", fileName)); if err != nil {
+	content, err := ioutil.ReadFile(fmt.Sprintf("type_fixtures/%s", fileName))
+	if err != nil {
 		log.Fatalf("Failed to load type fixture with file name '%s': %v", fileName, err)
 	}
 	return string(content)
```

Reply:
FWIW you probably want to squash down to a single commit

Reply:
Done.

Reply:
Thanks! Should be a quick merge now :) 🚀